Source/Core/DolphinQt/Debugger/CodeViewWidget.cpp

```diff
@@ -331,7 +331,19 @@ void CodeViewWidget::Update(const Core::CPUThreadGuard* guard)
 
     std::string ins = (split == std::string::npos ? disas : disas.substr(0, split));
     std::string param = (split == std::string::npos ? "" : disas.substr(split + 1));
-    const std::string_view desc = debug_interface.GetDescription(addr);
+    std::string desc;
+    int color = 0xFFFFFF;
+    const Common::Note* note = m_ppc_symbol_db.GetNoteFromAddr(addr);
+    if (note == nullptr)
+    {
+      desc = debug_interface.GetDescription(addr);
+      color = debug_interface.GetColor(guard, addr);
+    }
+    else
+    {
+      desc = note->name;
+      color = debug_interface.GetNoteColor(guard, addr);
```

Comment:
This might be misleading as there is no easy way to distinguish a symbol and a note. Can't you add a prefix allowing to distinguish a note from a regular symbol?

For instance, what about using `*` (based on depth) and space before displaying the note. For instance:
```
zz_normal_symbol
* Note without overlap
** Overlapping note
*** One more depth
```

I'm open to other alternatives if you have any.

Reply:
I was hoping just having a different color in the middle of a function would be enough, but I can see there are times when that won't be the case. Beyond that, user naming conventions might differentiate them. 

I don't think it's important to show exactly how many layers there are. "* note" is okay if that seems like it'd help with clarity.